src/ColumnsPaginatedBookView.ts

```diff
@@ -19,7 +20,7 @@ export default class ColumnsPaginatedBookView implements PaginatedBookView {
 
         // any is necessary because CSSStyleDeclaration type does not include
         // all the vendor-prefixed attributes.
-        const body = this.bookElement.contentDocument.body as any;
+        const body = HTMLUtilities.findRequiredElement(this.bookElement.contentDocument, "body") as any;
```

Comment:
I wish I could do this but HTMLElement doesn't have vendor-prefixed style attributes either. The best I could do would be to cast body.style to any instead of body.